docs/faq.md

```diff
@@ -0,0 +1,204 @@
+# .NET Standard FAQ
+
+*Something missing? [File an issue][issue].*
+
+## What are good resources to learn about .NET Standard?
+
+* [Introducing .NET Standard][netstandard-post] blog post
+* [Intro to .NET Standard](https://channel9.msdn.com/Blogs/Seth-Juarez/What-is-NET-Standard) video
+* [Q&A](https://channel9.msdn.com/Shows/On-NET/Immo-Landwerth-Net-Standard) video
+
+## What is .NET Standard?
+
+.NET Standard is a specification that represents a set of APIs that all .NET
+platforms have to implement. This unifies the .NET platforms and prevents future
+fragmentation. Think of .NET Standard as POSIX for .NET.
+
+Having a standard solves the code sharing problem for .NET developers by
+bringing all the APIs that you expect and love across the environments that you
+need: desktop applications, mobile apps & games, and cloud services.
+
+For more details, take a look at [Introducing .NET Standard][netstandard-post]
+blog post.
+
+## The blog post has 15 pages. Why so complicated?
+
+The general idea of .NET Standard is pretty simple indeed. The [blog
+post][netstandard-post] is a bit longer because it also provides more context in
+related areas, specifically how we use .NET Standard in tooling, which additions
+we're making in .NET Standard 2.0, how we model platform specific APIs, and what
+.NET Standard means for .NET Core.
+
+## I still don't get it. Can you provide an analogy that makes sense for a dev?
+
+David Fowler provided a [developer analogy][netstandard-analogy] that explains
+.NET Standard in terms of interfaces and classes.
+
+## How is .NET Standard different from .NET Core?
+
+Here is the difference:
+
+* .NET Standard is a specification that covers which APIs a .NET platform has to
+  implement.
+* .NET Core is a concrete .NET platform and implements the .NET Standard.
+
+## What APIs are part of .NET Standard and which platforms support it?
+
+We have a [version document][netstandard-versions] that points you to the
+platform support matrix as well as which APIs are available in a given .NET
+Standard version.
+
+## As a library author, which version of .NET Standard should I target?
+
+When choosing a [.NET Standard version][netstandard-versions] you should
+consider this trade-off:
+
+* The higher the version, the more APIs are available to you.
+* The lower the version, the more platforms you can run on.
+
+So generally speaking, you should target the lowest version you get away with.
+The [version document][netstandard-versions] will help inform your decision.
+
+## How does .NET Standard versioning work?
+
+Think of the .NET Standard versions as concentric circles: higher versions
+incorporate all APIs from previous versions.
```

Comment:
Now it does :-) See [here](https://github.com/dotnet/standard/blob/master/docs/netstandard-20/README.md#net-framework-461-supporting-net-standard-20).